docs/source/private_data_tutorial.rst

```diff
@@ -117,7 +110,7 @@ The data to be secured by these policies is mapped in chaincode and will be
 shown later in the tutorial.
 
 This collection definition file is deployed when the chaincode definition is
-committed to the channel using the `peer lifecycle chaincode commit command <http://hyperledger-fabric.readthedocs.io/en/latest/commands/peerchaincode.html#peer-chaincode-instantiate>`__.
+committed to the channel using the `peer lifecycle chaincode commit command <file:///Users/nikhilgupta/fabric/docs/build/html/commands/peerlifecycle.html#peer-lifecycle-chaincode-commit>`__.
```

Comment:
```suggestion
committed to the channel using the `peer lifecycle chaincode commit command <http://hyperledger-fabric.readthedocs.io/en/latest/commands/peerlifecycle.html#peer-lifecycle-chaincode-commit>`__.
```

Reply:
😬 Looks like I got the link from the wrong browser window. Thanks for the catch.